lib/data-types.js

```diff
@@ -352,6 +352,41 @@ BLOB.prototype.toSql = function() {
 };
 
 /**
+ * Range types are data types representing a range of values of some element type (called the range's subtype). Only available in postgres.
+ * See {@link http://www.postgresql.org/docs/9.4/static/rangetypes.html|Postgres documentation} for more details
+ * @property RANGE
+ */
+
+var RANGE = function (Subtype) {
+  var options = {};
+  if(typeof Subtype === 'function') { // if subtype passed - instantiate object of this subtype and return new function
+    options.subtype = new Subtype();
+    return RANGE.bind({}, options);
+  }
+  else if(typeof Subtype === 'object' && Subtype.hasOwnProperty('subtype'))
+    options = Subtype;
+
+  if (!(this instanceof RANGE)) return new RANGE(options);
+  ABSTRACT.apply(this, arguments);
+
+  this._subtype = options.subtype ? (options.subtype.key || 'INTEGER') : 'INTEGER';
```

Comment:
Please refactor to use INTEGER as default subtype if not present, and then just have this line be `options.subtype.key`